epochStart/bootstrap/process.go

```diff
@@ -1099,6 +1099,7 @@ func (e *epochStartBootstrap) createRequestHandler() error {
 
 	storageService := disabled.NewChainStorer()
 
+	log.Debug("epochStartBootstrap.createRequestHandler", "shard", e.shardCoordinator.SelfId())
```

Comment:
the request handlers - as metaResolversContainers should be used for Validator sync - epochstart metablocks. After that the node should know in which shard it is and have proper request handlers. With this, the node knows it is in shard 2 for example, but it is still pretending on being in meta.

Reply:
maybe add a TODO, a refactor here.

Reply:
added TODO